openSpotServer/core/views.py

```diff
@@ -17,9 +17,16 @@ def home(request):
 
 
 def custom_save(image_form):
```

Comment:
This function name is a little confusing. What are we doing just deleting the old image and saving the new one?

Reply:
changed name